### PR TITLE
On 3173/update with upstream master

### PR DESCRIPTION
**Pull request by leland-kwong**

### CODE REVIEW

**Review by olsonpm — APPROVED** [review]

**Comment by olsonpm** [comment]
@Leland-Kwong I think the flow for our clay-kiln fork can be to

1. merge into master-unity
2. publish
3. point unity to the new tag
4. then move the unity branch through the normal pr process.

This resembles how we'd treat updating any other dependency of unity.  My point being please merge this when you get a chance.  Don't worry about the publish step after you merge since I think the code should be essentially the same as what you already published

**Comment by leland-kwong** [comment]
Thanks, duly noted.

**Pull request merged**
